idaes/core/util/model_serializer.py

```diff
@@ -735,7 +733,7 @@ def to_json(
     # unfortunatly I can't write how long it took to write the file in the file
     pdict["etime_write_file"] = file_time - dict_time
     if return_dict:
-        # In interactive environments returning the dict can cuase it to print
+        # In interactive environments returning the dict can cause it to print
         # an extreemly large amount of stuff.  So added this option to make sure
```

Comment:
Typo: extremely

Reply:
Aw dang.  I need to look for a comment/string spell checker